addons/binding/org.openhab.binding.network.test/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It could be an openHAB issue, I noticed recently that `mvn test` did not do any tests when the build broke afer merging something which looked all green on travis.

It would be great if you could test it quickly by breaking one of your esh tests and running the test command. If it does not work for esh as well it would be great if you could great issue there otherwise create an issue in openHAB core and mention me.